docs/_includes/migrating-from-asciidoc.adoc

```diff
@@ -0,0 +1,353 @@
+// Issue #480. #434, #463 refer.
+// Conventions:
+// directory (not folder)
+// set/unset attributes (not define)
+// US spelling (single t in formating, ize not ise, color).
+// formulas (not formulae)
+// e.g. not eg
+:adp: Asciidoc Python
+:adr: Asciidoctor
+:url-test: https://github.com/asciidoctor/asciidoctor/
+:url-diagram: https://github.com/asciidoctor/asciidoctor-diagram/
+:url-migrate: http://asciidoctor.org/docs/migration/
+:url-recommended: http://asciidoctor.org/docs/asciidoc-recommended-practices/
+:url-diffs: http://asciidoctor.org/docs/asciidoc-asciidoctor-diffs/
```

Comment:
Can you use the prefix "uri-" instead of "url-"? That's the pattern we've been using. We'll probably want to promote these URLs to the user-manual.adoc (the master) as well, but for now we can keep them here.